state/session/__tests__/selectors.test.js

```diff
@@ -0,0 +1,29 @@
+import {
+  getSessionId,
+  isUserLoggedIn,
+  username as usernameSelector
+} from '../selectors';
+
+describe('Movie selectors.', () => {
+  const sessionId = '1q2w3e4r5t';
+  const username = 'mock-name';
+  const sessionIdBool = !!sessionId;
+  const state = {
+    session: {
+      sessionId,
+      username
+    }
+  };
+
+  it('getSessionId() returns sesion id', () => {
+    expect(getSessionId(state)).toEqual(sessionId);
+  });
+
+  it('isUserLoggedIn() converts sessionId to bool type and returns it', () => {
+    expect(isUserLoggedIn(state)).toEqual(sessionIdBool);
```

Comment:
You can use .toBeTruthy()
https://jestjs.io/docs/en/expect#tobetruthy

Reply:
Fixed